src/sql/workbench/api/browser/mainThreadConnectionManagement.ts

```diff
@@ -93,7 +92,7 @@ export class MainThreadConnectionManagement extends Disposable implements MainTh
 	}
 
 	public $getConnections(activeConnectionsOnly?: boolean): Thenable<azdata.connection.ConnectionProfile[]> {
-		return Promise.resolve(this._connectionManagementService.getConnections(activeConnectionsOnly).map(profile => convertToRpcConnectionProfile(profile, true, this._connectionManagementService.removeConnectionProfileCredentials)));
+		return Promise.resolve(this._connectionManagementService.getConnections(activeConnectionsOnly).map(profile => this.convertToConnectionProfile(profile)));
```

Comment:
@smartguest Please follow up with removing the deepCopyOptions and removeFunction parameters from the helper function since those aren't used currently (and the second is problematic as shown here)

In the future stuff like this is why you should pass the entire service as the parameter - not just a function. (passing functions around in general is dangerous since they lose all this context so you have to be careful to capture stuff correctly or it may not work as expected)